assets/js/vue/components/info-search/ResultSet.vue

```diff
@@ -14,7 +14,7 @@
             <!-- Need response data to get facet data -->
 <!--            <Facet :facet-info="results.facetInfo.productTypeDesc" :facet-name="facetLabels.productTypeDesc" v-on="$listeners" :formValues="formValues"/>-->
 <!--            <Facet :facet-info="results.facetInfo.digitalTypeDesc" :facet-name="facetLabels.digitalTypeDesc" v-on="$listeners" :formValues="formValues"/>-->
-<!--            <Facet :facet-info="results.facetInfo.researchGroupsInfo" :facet-name="facetLabels.researchGroup" v-on="$listeners" :formValues="formValues"/>-->
+           <Facet :facet-info="results.facetInfo.researchGroupsInfo" :facet-name="facetLabels.researchGroup" v-on="$listeners" :formValues="formValues"/>
```

Comment:
It is variable issue, the api was returning `researchGroupInfo`, made that change in my branch and some other required changes. Please undo this change and pull my branch, then we can uncomment this line and other facets too